app/views/devise/registrations/edit.html.erb

```diff
@@ -0,0 +1,43 @@
+<h2>Edit <%= resource_name.to_s.humanize %></h2>
```

Comment:
Is this form currently being used? If not remove it for now.
I know it was generated by devise, but don't include files you're not using.
If you really want to keep it around, put it on another branch.